tools/purge_ems.rb

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
+enviroment = File.expand_path('../config/environment', __dir__) + ".rb"
+require enviroment if File.exist?(enviroment)
+
+if !defined?(Vmdb::Application)
+  warn "This script must exist in the tools directory or can live elsewhere but needs to be run via rails runner path/to/purge_ems.rb --id 1234 --follow true"
+  exit 1
+end
+
+require 'optimist'
+
+ARGV.shift if ARGV[0] == '--' # if invoked with rails runner
+opts = Optimist.options do
+  banner "Purge managements system records in the background via the queue.\n\nUsage: ruby #{$0} [options]\n\nOptions:\n\t"
+  opt :id,      "Mangement System id",                                         :short => :i, :type => :integer,                    :required => true
+  opt :follow,  "Follow the progress or exit after creating messages",         :short => :f, :type => :string,  :default => "true"
+  opt :batch,   "How many rows in each relationship to be deleted in a batch", :short => :b, :type => :integer, :default => 1_000
+  opt :timeout, "How many minutes should each message be allowed to run",      :short => :t, :type => :integer, :default => 30
```

Comment:
Ah yes.  I started with --watch (instead of follow) and --window (instead of batch).  I prefer these names and yeah, I think it's automatic if they begin with different letters.